backend/pyproject.toml

```diff
@@ -0,0 +1,103 @@
+[project]
+version = "2.5.8"
+name = "ibutsu_server"
+description = "A system to store and query test results and artifacts"
+authors = [
+    {name = "Raoul Snyman"},
+    {name = "John Dupuy"},
+]
+maintainers = [
+    {name = "Mike Shriver", email = "mshriver@redhat.com"},
```

Comment:
@RonnyPfannschmidt @jjaquish @LightOfHeaven1994  (and anyone else that sees this)

Would you like to be listed here? Red Hat emails?

Reply:
-1 for me

Reply:
@mshriver yep, add me please :pray: